amazon-arm64.pkr.hcl

```diff
@@ -0,0 +1,205 @@
+variable "ami" {
+  type    = string
+  default = "ubuntu/images/hvm-ssd/ubuntu-focal-20.04-arm64-server-*"
+}
+
+variable "ami_name" {
+  type    = string
+  default = "supabase-postgres-arm-13.3.0.4"
+}
+
+variable "ami_regions" {
+  type    = list(string)
+  default = ["us-west-2"]
```

Comment:
either remove, or default to all regions?

Reply:
removed this too.